packages/@aws-cdk/aws-cloud9/test/cloud9.environment.test.ts

```diff
@@ -66,4 +67,45 @@ test('throw error when subnetSelection not specified and the provided VPC has no
       instanceType: ec2.InstanceType.of(ec2.InstanceClass.C5, ec2.InstanceSize.LARGE),
     });
   }).toThrow(/no subnetSelection specified and no public subnet found in the vpc, please specify subnetSelection/);
+});
+
+test('imoprt codecommit repo', () => {
```

Comment:
This test name doesn't seem great... How about something like `can use CodeCommit repositories`?